lib/cucumber/cli/options.rb

```diff
@@ -24,7 +24,7 @@ class Options
                                                               "#{INDENT}the usage formatter, except that steps are not printed."],
         'junit'       => ['Cucumber::Formatter::Junit',       "Generates a report similar to Ant+JUnit. Use\n" \
                                                               "#{INDENT}junit,fileattribute=true to include a file attribute."],
-        'json'        => ['Cucumber::Formatter::Json',        '[DEPRECATED] Prints the feature as JSON'],
+        'json'        => ['Cucumber::Formatter::Json',        'Prints the feature as JSON'],
```

Comment:
I meant for example on this line, could we say something like (Does not use messages)

Again, if you feel this needs a wider discussion, maybe at Cukenfest tomorrow I can do.

Reply:
I don't think the users will be any wiser if we say what you suggest @luke-hill. I propose:

```
Prints the feature as JSON. The JSON format is in maintenance mode. Please consider using the message formatter instead.
```

Reply:
I would go further with something like:
`Prints the feature as JSON. The JSON format is in maintenance mode. Please consider using the message formatter with the standalone json-formatter instead (https://github.com/cucumber/cucumber/tree/master/json-formatter).`

I think that someone with the need for formatting as json has not the same needs as someone which would benefit from the messages directly.

Also we could add later a dedicated webpage for the json-formatter somewhere

Reply:
Go for it. Both those options are 10* better than mine